mobile/src/navigations/chat/chat.tsx

```diff
@@ -0,0 +1,23 @@
+import { createNativeStackNavigator } from '@react-navigation/native-stack';
+import React from 'react';
+
+import { ChatScreenName } from '#libs/enums/enums';
+import { type ChatNavigationParameterList } from '#libs/types/types';
+import { Chat } from '#screens/chat/chat';
+import { ChatList } from '#screens/chat-list/chat-list';
+
+const NativeStack = createNativeStackNavigator<ChatNavigationParameterList>();
+
+const ChatNavigation: React.FC = () => {
```

Comment:
```suggestion
const Chat: React.FC = () => {
```

Reply:
But I already have a component with 'Chat' name (screens/chat/chat)

Reply:
let's rename import to something like
`import { Chat as ChatPanel }`

Reply:
Yes, sorry about that. I was trying to do it that way, but I encountered some strange errors, and I thought that React Native might not allow having two components with the same name. Right now, I'm not getting those errors. Thanks